utils.py

```diff
@@ -48,3 +50,9 @@ def copy_latest_continuous_monitor(hostname, identity_file, filename_prefix, use
   plot_continuous_monitor.plot_continuous_monitor(local_continuous_monitor_file, open_graphs=True)
   return local_continuous_monitor_file
 
+def __plot_continuous_monitors(log_dir):
```

Comment:
shouldn't be underscores here (which are intended for private functions that shouldn't be used elsewhere)